src/routes/v1/index.js

```diff
@@ -0,0 +1,8 @@
+import { Router } from 'express';
+import translatorController from '../../controllers/v1/translator';
+
+const apiRouter = Router();
+
+apiRouter.post('/translate', translatorController);
```

Comment:
Should this be a post? 🤔

Reply:
Since there are no side effects caused by using this route, I have changed this to a `GET` route.